majilove/models.py

```diff
@@ -102,7 +102,7 @@ def display_blood_type(self):
 
     image = models.ImageField(_('Image'), upload_to=uploadItem('idol'))
 
-    small_image = models.ImageField(_('Image'), upload_to=uploadItem('idol/small'))
+    small_image = models.ImageField(string_concat(_('Image'), ' (', _('Small'), ')'), upload_to=uploadItem('idol/small'))
```

Comment:
Is this image only going to be used for the map? If so, the label will never be seen by users so you don't need to make the label translatable (`models.ImageField('Small image (for map)', ...)`.